addons/weapons/sx90/CfgMagazines.inc

```diff
@@ -0,0 +1,33 @@
+#define CLASSES_MAGAZINES \
+  "SWS_Magazine_50rnd_SX90" \
+  , "SWS_Magazine_50rnd_SX90_Tracers"        \
+  , "SWS_Magazine_50rnd_SX90_Tracers_Blue"   \
+  , "SWS_Magazine_50rnd_SX90_Tracers_Yellow"
+
+class CfgMagazines
+{
+  class 50Rnd_570x28_SMG_03;
+  class SWS_Magazine_50rnd_SX90 : 50Rnd_570x28_SMG_03
+  {
+    ITEM_META(2);
+    CLEARANCE(RESTRICTED/DECWHI);
+    ammo = "B_65x39_Caseless";
+    displayName = QNAME(50Rnd 6.5mm Sx90 Magazine);
+    displayNameShort = QUOTE(6.5mm Caseless);
+    descriptionShort = "Caliber: 6.5mm<br />Rounds: 50<br />Used in: SX90";
+    mass = 17;
+    lastRoundsTracer = 5;
+  };
+  class SWS_Magazine_50rnd_SX90_Tracers : SWS_Magazine_50rnd_SX90 {
+    displayName = QNAME(50Rnd 6.5mm Sx90 Mag (Red Tracer));
+    tracersEvery = 1;
+  };
+  #define C_TRACERS(varColor) \
+  class SWS_Magazine_50rnd_SX90_Tracers_##varColor : SWS_Magazine_50rnd_SX90_Tracers \
+  { displayName = QNAME(50Rnd 6.5mm Sx90 Mag (varColor Tracer)); ammo = QUOTE(B_65x39_Caseless_##varColor); }
+
+  C_TRACERS(Blue);
```

Comment:
When using blue tracers, make sure you mark `sws_ballistics_tracers` as a dependency in the config.cpp.